src/components/Search/searchAutoFillComponent.js

```diff
@@ -13,7 +13,7 @@ function searchComponent() {
   const [inputValue, setInputValue] = React.useState('');
   const [options, setOptions] = React.useState([]);
   const loading = open;
-
+  
```

Comment:
Codacy has a fix for the issue: [Trailing spaces not allowed.](https://app.codacy.com/gh/CBIIT/bento-frontend/pullRequest?prid=8502143)
```suggestion

```